packages/web/src/components/checklist/common/LocalTextAdapter.js

```
@@ -0,0 +1,212 @@
+  insert(index, text) {
+    const before = this._value.substring(0, index);
+    const after = this._value.substring(index);
+    this._value = before + text + after;
+    this._notifyUpdate();
+  }
+
+  _notifyUpdate() {
+    if (this._onUpdate) {
+      this._onUpdate(this._value);
+    }
+    this._observers.forEach(callback => callback());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)